gobblin-service/src/main/java/org/apache/gobblin/service/modules/orchestration/DagProcessingEngine.java

```diff
@@ -149,6 +158,17 @@ public void run() {
           dagTask.conclude();
           log.info(dagProc.contextualizeStatus("concluded dagTask"));
         } catch (Exception e) {
+          if(!DagProcessingEngine.isTransientException(e)) {
+            DagActionStore.DagAction dagAction = dagTask.getDagAction();
+            if(dagAction!=null) {
+              log.error(
+                  "Ignoring non transient exception. DagTask with dagId: {} and dagAction: {} will conclude and will not be retried.",
+                  dagAction.getDagId(), dagAction.getDagActionType(), e);
+            }
```

Comment:
1. replace L162-167, w/ equivalent:
```
log.error(dagProc.contextualizeStatus("ignoring non-transient exception by concluding so no retries"));
```
2. let's move general announcement of an the error before special handling of non-transient, specifically:
```
log.error("DagProcEngineThread: " + dagProc.contextualizeStatus("error"), e);
```
3. in the above (2.) log it as `error` and keep the exception as the 2nd arg, but the message about ignoring can probably just be `warn` and needn't repeat the same stacktrace (so no exception as 2nd arg).

4 and although it may go away, pointing out that the way to format:
```
if(dagAction!=null) {
```
is w/ whitespace
```
if (dagAction != null) {
```

Reply:
sure, updated